frontend/src/bundles/overview/components/chart-goal-progress/chart-goal-progress.tsx

```diff
@@ -0,0 +1,234 @@
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday',
+    ],
+    datasets: [
+        {
+            label: 'Workout',
+            data: [20, 30, 40, 50, 60, 70, 80],
+            backgroundColor: colors.workout.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+        {
+            label: 'Calories',
+            data: [50, 60, 70, 80, 90, 100, 11],
+            backgroundColor: colors.calories.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+        {
+            label: 'Steps',
+            data: [100, 20, 30, 40, 50, 60, 70],
+            backgroundColor: colors.steps.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+    ],
+};
+
+const monthlyData = {
+    labels: ['Week 1', 'Week 2', 'Week 3', 'Week 4'],
+    datasets: [
+        {
+            label: 'Workout',
+            data: [20, 30, 40, 50],
+            backgroundColor: colors.workout.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+        {
+            label: 'Calories',
+            data: [50, 60, 70, 80],
+            backgroundColor: colors.calories.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+        {
+            label: 'Steps',
+            data: [100, 20, 30, 40],
+            backgroundColor: colors.steps.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+    ],
+};
+
+const yearlyData = {
+    labels: [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+    ],
+    datasets: [
+        {
+            label: 'Workout',
+            data: [20, 30, 40, 50, 60, 70, 80, 90, 100, 70, 50, 55],
+            backgroundColor: colors.workout.backgroundColor,
+            borderColor: 'transparent',
+            margin: 10,
+        },
+        {
+            label: 'Calories',
+            data: [50, 60, 70, 80, 90, 100, 100, 20, 30, 40, 50, 16],
+            backgroundColor: colors.calories.backgroundColor,
+
+            borderColor: 'transparent',
+        },
+        {
+            label: 'Steps',
+            data: [10, 20, 30, 40, 0, 60, 70, 8, 90, 100, 11, 12],
+            backgroundColor: colors.steps.backgroundColor,
+            borderColor: 'transparent',
+        },
+    ],
+};
+
+const ChartGoalProgress = (): JSX.Element => {
+    const [currentData, setCurrentData] = useState<SelectOption>({
+        value: 'weekly',
+        label: 'Weekly',
+    });
+
+    const optionsBarChart = {
+        scales: {
+            y: {
+                beginAtZero: true,
+                ticks: {
+                    min: 0,
+                    max: 100,
+                    stepSize: 20,
+                    callback: function (value: number): string {
+                        return `${value}%`;
+                    },
```

Comment:
```suggestion
                    callback: (value: number): string => `${value}%`;,
```
Is it possible to do like this?